doc/realtime_servo/launch/servo_server_panda.launch.py

```diff
@@ -117,8 +117,8 @@ def generate_launch_description():
             ),
             ComposableNode(
                 package="moveit_servo",
-                plugin="moveit_servo::ServoServer",
-                name="servo_server",
+                plugin="moveit_servo::ServoNode",
```

Comment:
#113 deleted this launch file since it is duplicated in the main package. #140 erroneously added it back in (oops). Suggest deleting it again.